test/system/test_tpm2_pcrevent.sh

```diff
@@ -58,10 +58,18 @@ tpm2_pcrevent -Q < $hash_in_file
 
 # Test that fifo stdin works
 cat $hash_in_file | tpm2_pcrevent > $hash_out_file
+cat $hash_out_file
 
 # Verify output as expected.
 for l in `cat $hash_out_file`; do
   alg=`echo -n $l | cut -d\: -f 1-1`
+  if [ x"$alg" == x"sha" ]; then
+    alg="sha1"
```

Comment:
@williamcroberts this is in fact a bug in the tpm2_alg_util_algtostr() function.

The spec makes clear that both TPM_ALG_SHA and TPM_ALG_SHA1 refer to the SHA-1 algorithm, but the function table has entries for both even when map to the same hex value (0x0004). So instead I believe that TPM_ALG_SHA shouldn't be defined and always map 0x0004 to "sha1".

Otherwise you can't really convert TPM_ALG_SHA1 to its mapped string since TPM_ALG_SHA will always match first:

```
tpm2_tool_output("hash algorithm is %s\n", tpm2_alg_util_algtostr(TPM_ALG_SHA1));

hash algorithm is sha
```

Reply:
This crossed my mind yesterday. Yeah I could see the argument both ways, since they both map to the same numerical value, should sha and sha1 exist as friendly identifiers? I could see a case for both arguments, but today I am leaning towards killing sha. Let's take PR https://github.com/01org/tpm2-tools/pull/526 for the algutil changes and then you can update this PR with just the tpm2_tool_output() changes.